test/js-api/wasm-module-builder.js

```diff
@@ -148,7 +150,7 @@ class WasmModuleBuilder {
     return this;
   }
 
-  addMemory(min, max, exp) {
+  addMemory(min, max, exp, shared = false) {
```

Comment:
I intentionally kept the `shared` param because it's passed in `limits.any.js` test suite, but it's not emited.

Reply:
Yes, that's a good point. Some changes are needed in the `limits.any.js` test suite so I can remove it here and in another PR its usage.